packages/ui/src/common/hooks/useRefetchQueries.ts

```diff
@@ -38,6 +38,7 @@ export const useRefetchQueries = (
         }, interval)
 
         return () => {
+          isRefetched.current = false
```

Comment:
@IlyaSmiyukha I don't think this change is right. AFAIK `isRefetched` is just supposed to indicate that the data was already fetched once. So once it has been set to `true` it should remain this way. Or am I missing something ?

Reply:
For the current post you are right, But when the user opens another post `isRefetched` will return true. And in [this case ](https://github.com/Joystream/pioneer/blob/e26105af6e27c36142c3ef748c00cd85ea057dc6/packages/ui/src/app/pages/Forum/ForumThread.tsx#L38) he will be redirected to 404 page. So when the URL was changed we should update isRefetched as well